substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jfr/SubstrateJVM.java

```diff
@@ -530,6 +532,12 @@ public boolean setCutoff(long eventTypeId, long cutoffTicks) {
         return true;
     }
 
+    /** See {@link JVM#setThrottle}. */
+    public boolean setThrottle(long eventTypeId, long eventSampleSize, long periodMs) {
+        Logger.log(LogTag.JFR_SYSTEM_THROTTLE, LogLevel.WARN, "JFR event throttling is not yet supported in native images.");
```

Comment:
This is not going to work on JDK11, but I don't think we need to introduce a JDK17+-specific package for this. I'm going to fix this this in https://github.com/oracle/graal/pull/4896.